Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -1791,6 +1791,15 @@
                     }
                 }
             },
+            "TopicColor": {
+                "type": "object",
+                "properties": {
+                    "standardColorIdentifier": {
```

Comment:
Not a blocker but should the name of this field more closely resemble the semantic name used in theme-settings?

For example, the theme-settings name for red elsewhere in the PR is `standard-red-documentation-intro-fill` but it gets emitted in render JSON here as `standardColorIdentifier: "red"`

Or maybe I'm misunderstanding and it's meant to be more generic here? Just curious.

Reply:
That's intentional for now. The goal is to keep Swift-DocC as generic as possible and then whatever renderer is being used would allow for the more specific customization of what `red` means.

We'll need to revisit this when we actually add more customization, but the idea is that this allows for a world where we have:

```
standard-red-documentation-intro-fill
standard-red-documentation-link-highlight
standard-red-documentation-intro-icon
```

which would all be individually customizable in theme-settings. An author would then opt in to using those colors on a specific page by just specifying `red` as the standard color for the page.